README.md

```diff
@@ -0,0 +1,197 @@
+# Final group capstone - Book an Appointment
+
+<a name="readme-top"></a>
```

Comment:
- [ ] Kindly, include in your instructions the required db commands that your users will need to run your backend app: ``rails db:create, rails db:migrate, rails db:seed``. 👍